ando/tests/test_Rules.py

```diff
@@ -48,6 +48,15 @@ def test_subject_rules_2(self):
         list = ["Sub_001"]
         self.assertEqual(andoE.is_subject(list), False)
 
+    '''testing ephys rules'''
+    def test_ephys_rules(self):
+        list = ["ephys"]
+        self.assertEqual(andoE.is_ephys(list), True)
+    def test_ephys_rules1(self):
+        list = ["Ephys"]
+        self.assertEqual(andoE.is_ephys(list), False)
```

Comment:
maybe add one test with a string that just contains 'ephys' but also other characters?

Reply:
Yep that's a good Idea.